app/assets/stylesheets/custom.scss

```diff
@@ -21,3 +21,15 @@ abbr[title="required"] {
   margin-bottom: 0;
 }
 
+.navbar-default .navbar-text {
+  color: #fff;
+}
+
+/* nav */
+
+// This removes the hover and all other events from a link - used for name element
+
```

Comment:
Perhaps remove this space so the comment is visually more bound to its related CSS.